rlclientlib/logger/http_transport_client.h

```diff
@@ -89,4 +99,137 @@ namespace reinforcement_learning {
     i_trace* _trace;
     error_callback_fn* _error_callback;
   };
+
+  template <typename TAuthorization>
```

Comment:
There are benefits of separating definition and implementation, but it is probably more consistent with the code base to keep them together. It also helps to get rid of "template <typename TAuthorization>" copy-paste.

Reply:
Leaving the entire implementation inline means you cannot look at the class definition to see what the interface is without reading past many lines of implementation code though?

Reply:
yeah I agree with this. My only argument was consistency, but actually you are right - we are not inlining it.